Sources/SwiftCurrent/Models/Workflow.swift

```diff
@@ -94,7 +94,7 @@ public final class Workflow<F: FlowRepresentable>: LinkedList<_WorkflowItem> {
      - Parameter orchestrationResponder: the `OrchestrationResponder` to notify when the `Workflow` proceeds or backs up.
      - Parameter passedArgs: the arguments to pass to the first instance(s).
      - Parameter launchStyle: the launch style to use.
-     - Parameter onFinish: the closure to call when the last element in the workflow proceeds; called with the `AnyWorkflow.PassedArgs` the workflow finished with.
+     - Parameter onFinish: the closure to call when the last element in the workflow proceeds; called with the `AnyWorkflow.PassedArgs` of the last node in the workflow.
```

Comment:
This is no longer a true statement. I doesn't end with a preposition, but it's false.